File: ru.kmorozov.gbd.core.logger/src/main/java/ru/kmorozov/gbd/logger/listeners/AbstractLogEventListener.java
package ru.kmorozov.gbd.logger.listeners;

import ru.kmorozov.gbd.logger.events.BaseEvent;
import ru.kmorozov.gbd.logger.events.LogEvent;

/**
 * Created by km on 15.12.2015.
 */
public abstract class AbstractLogEventListener implements IEventListener {

    @Override
    public boolean eventMatched(BaseEvent event) {
        return event instanceof LogEvent;
    }
}

File: ru.kmorozov.gbd.core.logger/src/main/java/ru/kmorozov/gbd/logger/listeners/AbstractLogEventListener.kt
package ru.kmorozov.gbd.logger.listeners

import ru.kmorozov.gbd.logger.events.LogEvent

/**
 * Created by km on 15.12.2015.
 */
abstract class AbstractLogEventListener : IEventListener {

    override fun eventMatched(event: LogEvent): Boolean {
        return true
    }
}
